examples/notebooks/plot_grid_search_census.py

```diff
@@ -42,7 +41,16 @@
 # %%
 # We can now load and inspect the data by using the `fairlearn.datasets` module:
 
-data = fetch_adult(as_frame=True)
+import pathlib
+from sklearn.datasets import fetch_openml
+data = fetch_openml(
+    data_id=1590,
+    data_home=pathlib.Path().home() / ".fairlearn-data",
+    cache=True,
+    as_frame=True)
+# In the future this will be available as follows:
+# from fairlearn.datasets import fetch_adult
+# data = fetch_adult(as_frame=True)
```

Comment:
similarly, please get rid of the lines 44, 48, 49. no need for importing "pathlib" and providing "data_home" and "cache".